runners/flink/src/main/java/org/apache/beam/runners/flink/FlinkPipelineOptions.java

```diff
@@ -142,6 +142,20 @@ public interface FlinkPipelineOptions
 
   void setNumberOfExecutionRetries(Integer retries);
 
+  @Description(
+      "Set job check interval in seconds under detached mode in method waitUntilFinish, "
+          + "by default it is 5 seconds")
+  @Default.Integer(5)
+  int getJobCheckIntervalInSecs();
+
+  void setJobCheckIntervalInSecs(int seconds);
+
+  @Description("Set the attached mode")
```

Comment:
nit: probably can re-phrase the comment as "Specifies if the pipeline is submitted in attached or detached mode"

Reply:
Thanks for the review @kkdoon I have updated the comment.